train_network.py

```diff
@@ -0,0 +1,179 @@
+import tensorflow as tf
+import numpy as np
+from accuracy import *
```

Comment:
`import *` not generally recommended. I doubt there are that many things to import.